mpqp/core/circuit.py

```diff
@@ -222,6 +224,9 @@ def add(self, components: OneOrMany[Instruction | NoiseModel]):
             components.size = self.nb_qubits
 
         if isinstance(components, NoiseModel):
+            if len(components.targets) == 0:
+                components.targets = [target for target in range(self.nb_qubits)]
```

Comment:
This comes from a previous commit on dev, we said that the fact that target is empty will integrated by merging the other branch related to this feature. So this part should be removed

Reply:
this code actually doesn't come from me. It's only marked new because I merged the branch it comes from with mine to check compatibility

Reply:
but it was a merge error, I'll remove it, thanks for the reminder :)